pontoon/base/models.py

```diff
@@ -180,6 +183,35 @@ def user_translated_locales(self):
 
     return [locale.code for locale in locales]
 
+
+@property
+def user_rights(self):
+    output = 'Contributor'
+
+    if self.translated_locales:
+        if self.is_superuser:
+            return 'Admin'
+
+        output = ''
+        managed = []
+        translated = []
```

Comment:
`translated` is basically `self.translated_locales`, so we don't need a separate variable.

Also, we should create `self.managed_locales` (check [this](https://github.com/mozilla/pontoon/blob/master/pontoon/base/models.py#L178) out ), so we don't need `managed`.

Once you have that, all you need to do is:
```
if self.is_superuser:
  return 'Admin'
if self.managed_locales:
  return 'Manager for ' + ', '.join(self.managed_locales)
if self.translated_locales:
  return 'Translator for ' + ', '.join(self.managed_locales)
else:
  return 'Contributor'
```